tests/subsys/net/lib/lwm2m_client_utils/testcase.yaml

```diff
@@ -1,6 +1,7 @@
 tests:
   subsys.net.lib.lwm2m_client_utils.unittest:
     tags: unittest
-    platform_allow: native_posix
+    platform_allow: native_posix qemu_x86
```

Comment:
Why allow more platforms for unittests? None of these should be platform specific, so this just wastes CI time.